client/my-sites/stats/post-performance/index.jsx

```diff
@@ -70,14 +73,12 @@ const StatsPostPerformance = React.createClass( {
 	},
 
 	render() {
-		const { site, query, post, loading } = this.props;
+		const { site, query, post, isRequesting } = this.props;
 		const postTime = post ? this.moment( post.date ) : this.moment();
-		const cardClass = classNames( 'stats-module', 'stats__latest-post-summary', 'is-site-overview', {
-			'is-loading': loading,
-			'is-hidden': ! loading && ! post
-		} );
+		const cardClass = classNames( 'stats-module', 'stats__latest-post-summary', 'is-site-overview' );
```

Comment:
I think retaining the loading state/class here is still worthwhile to show that a network request is in-progress.